libsolidity/ast/ASTJsonImporter.h

```diff
@@ -0,0 +1,118 @@
+	ASTPointer<EnumDefinition> createEnumDefinition(Json::Value const& _node);
+	ASTPointer<EnumValue> createEnumValue(Json::Value const& _node);
+	ASTPointer<ParameterList> createParameterList(Json::Value const& _node);
+	ASTPointer<FunctionDefinition> createFunctionDefinition(Json::Value const& _node);
+	ASTPointer<VariableDeclaration> createVariableDeclaration(Json::Value const& _node);
+	ASTPointer<ModifierDefinition> createModifierDefinition(Json::Value const& _node);
+	ASTPointer<ModifierInvocation> createModifierInvocation(Json::Value const& _node);
+	ASTPointer<EventDefinition> createEventDefinition(Json::Value const& _node);
+	ASTPointer<ElementaryTypeName> createElementaryTypeName(Json::Value const& _node);
+	ASTPointer<UserDefinedTypeName> createUserDefinedTypeName(Json::Value const& _node);
+	ASTPointer<FunctionTypeName> createFunctionTypeName(Json::Value const& _node);
+	ASTPointer<Mapping> createMapping(Json::Value const& _node);
+	ASTPointer<ArrayTypeName> createArrayTypeName(Json::Value const& _node);
+//	ASTPointer<InlineAssembly> createInlineAssembly(Json::Value const& _node);
+	ASTPointer<Block> createBlock(Json::Value const& _node);
+	ASTPointer<PlaceholderStatement> createPlaceholderStatement(Json::Value const& _node);
+	ASTPointer<IfStatement> createIfStatement(Json::Value const& _node);
+	ASTPointer<WhileStatement> createWhileStatement(Json::Value const& _node, bool _isDoWhile);
+	ASTPointer<ForStatement> createForStatement(Json::Value const& _node);
+	ASTPointer<Continue> createContinue(Json::Value const& _node);
+	ASTPointer<Break> createBreak(Json::Value const& _node);
+	ASTPointer<Return> createReturn(Json::Value const& _node);
+	ASTPointer<Throw> createThrow(Json::Value const& _node);
+	ASTPointer<VariableDeclarationStatement> createVariableDeclarationStatement(Json::Value const& _node);
+	ASTPointer<ExpressionStatement> createExpressionStatement(Json::Value const& _node);
+	ASTPointer<Conditional> createConditional(Json::Value const& _node);
+	ASTPointer<Assignment> createAssignment(Json::Value const& _node);
+	ASTPointer<TupleExpression> createTupleExpression(Json::Value const& _node);
+	ASTPointer<UnaryOperation> createUnaryOperation(Json::Value const& _node);
+	ASTPointer<BinaryOperation> createBinaryOperation(Json::Value const& _node);
+	ASTPointer<FunctionCall> createFunctionCall(Json::Value const& _node);
+	ASTPointer<NewExpression> createNewExpression(Json::Value const& _node);
+	ASTPointer<MemberAccess> createMemberAccess(Json::Value const& _node);
+	ASTPointer<IndexAccess> createIndexAccess(Json::Value const& _node);
+	ASTPointer<Identifier> createIdentifier(SourceLocation location, std::string const& name);
+	ASTPointer<ElementaryTypeNameExpression> createElementaryTypeNameExpression(Json::Value const& _node);
+	ASTPointer<ASTNode> createLiteral(Json::Value const& _node);
+
+	template<class T>
+	ASTPointer<T> nullOrCast(Json::Value _json);
+	Declaration::Visibility getVisibility(Json::Value const& _node);
```

Comment:
`get` as a prefix is discouraged (I know it is present in lots of places, but I think it just adds verbosity).